src/Bottleneck.coffee

```diff
@@ -17,7 +17,11 @@ class Bottleneck
 		@reservoir = null
 		@limiter = null
 		@events = {}
-	_trigger: (name, args) -> setTimeout (=> @events[name]?.forEach (e) -> e.apply {}, args), 0
+	_trigger: (name, args) ->
+		if name == 'dropped' && @rejectOnDrop
```

Comment:
I don't really have a preference regarding single quotes vs double quotes. In JS I've entirely switched to single quotes, but in CS I still use double quotes. Regardless, I think projects should pick one and stick to it. Bottleneck uses double quotes.